app/src/main/java/com/framgia/englishconversation/screen/audiodetail/AudioDetailViewModel.java

```diff
@@ -6,6 +6,8 @@
 import android.os.Build;
 import android.support.v4.app.FragmentManager;
 import android.util.Log;
+import android.view.View;
+import android.view.ViewGroup;
```

Comment:
ERROR: Unused import: android.view.ViewGroup. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports